content/en/platform/corda/4.5/enterprise/operations/monitoring-logging/metrics-monitoring-scenarios.md

```diff
@@ -37,4 +37,4 @@ You can get metrics for your node from these key sources:
 * **P2P** - Messaging between parties can be measured in a number of ways, including metrics for latency between messages being sent and received between nodes, the size of sent messages, the interval between received P2P messages.
 * **Other metrics** - Measure the tine taken to sign a transaction or check whether proposed network parameter updates have been accepted yet.
 
-Take a look at the [Node metrics documentation](../../node/operating/node-metrics) for a complete range of the metrics data available from your node.
+Take a look at the [Node metrics documentation](../../node/operating/node-metrics.md) for a complete range of the metrics data available from your node.
```

Comment:
This link needs to point here: en/platform/corda/4.5/enterprise/node-metrics.html
so maybe:
```suggestion
Take a look at the [Node metrics documentation](../../node-metrics.md) for a complete range of the metrics data available from your node.
```